src/server/cray/cfs/api/controllers/components.py

```diff
@@ -152,8 +152,19 @@ def put_components():
 def patch_components():
```

Comment:
Do you need any unit tests for these controller methods?

Reply:
This is something we've talked about as a team, but never came to a consensus.  Some people argue they rarely catch problems and aren't worth it.  Regardless, we haven't worried about unit tests in a long time because we don't have the time.